gallery/src/index.html

```diff
@@ -13,8 +13,8 @@
     <title>Sonatype shared component gallery</title>
     <link rel="stylesheet" href="gallery.css"/>
   </head>
-  <body class="nx-body">
-    <div id="ui" class="nx-page"></div>
+  <body class="nx-body gallery-body">
+    <div id="ui" class="nx-page gallery-page"></div>
```

Comment:
Do these do anything?

Reply:
> Do these do anything?

No not anymore: https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/8b04e3d37a6048811c980e1b09b67487cfb320e7